src/controllers/accounts/AccountsAssetsController.ts

```diff
@@ -0,0 +1,65 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+import { BadRequest } from 'http-errors';
+
+import { validateAddress } from '../../middleware';
+import { AccountsAssetsService } from '../../services/accounts';
+import AbstractController from '../AbstractController';
+
+export default class AccountsAssetsController extends AbstractController<AccountsAssetsService> {
+	constructor(api: ApiPromise) {
+		super(api, '/accounts/:address/', new AccountsAssetsService(api));
+	}
+
+	protected initRoutes(): void {
+		this.router.use(this.path, validateAddress);
+
+		this.safeMountAsyncGetHandlers([
+			['asset-balances', this.getAssetBalances],
+			['asset-approvals', this.getAssetApprovals],
+		]);
+	}
+
+	private getAssetBalances: RequestHandler = async (
+		{ params: { address }, query: { at, assets } },
+		res
+	): Promise<void> => {
+		const hash = await this.getHashFromAt(at);
+
+		let assetsArray: number[] = [];
+		if (typeof assets === 'string') {
+			assetsArray = this.parseQueryParamArrayOrThrow(
+				assets,
+				'assets query parameter is not in the correct format.'
+			);
+		}
```

Comment:
```suggestion
		const assetsArray = Array.isArray(assets) 
		     ? this.parseQueryParamArrayOrThrow(
				assets,
				'assets query parameter is not in the correct format.'
			)
		    : [ ];	
```
1) Assumes when we update the query param syntax it gives us an array
2) Ternaries are fun! But I don't think its a must :)